examples/mixed-multigrid-solver/data/A.mtx

```diff
@@ -0,0 +1,114 @@
+%%MatrixMarket matrix coordinate integer symmetric
```

Comment:
This is the matrix used by many examples, I think?

Reply:
But you're just adding this new file now, right? Maybe you could use one of the existing files in `matrices/test/`?

Reply:
I think all example should be standalone. we only have one example use that, but others all create a new one.

Reply:
Won't that significantly increase the size of the repository for no good reason?

Reply:
It is something we need to pay for. The example should work if we just copy the one of example folder to anywhere and CMake can find the corresponding lib (Ginkgo, etc).
Maybe @tcojean or @upsj have some comment here?

Reply:
Yes that's a problem with the current example setup, currently if we want them to be standalone and for the user to easily replace data and play with it, we need to have the data in this fashion. It also helps because the path to the file is only relative so you need to launch it from the example directory itself.

We can make it so that we share the example data, and we probably should, but that would require a bit of work in all examples and maybe at the same time we could make the examples installable as well and use full paths for the matrix data. I think that should be another PR?

Reply:
Okay. We should look into that at some point.

Reply:
I think generating the matrix inside *each of the examples*, when possible is also an option. This would also make it easier to increase the problem size, and also not have the overhead of having to read matrices/worry about having the correct paths and so on.

Reply:
the generating matrix is hard to change cases. it only have the size choice